README.md

```diff
@@ -1,18 +1,116 @@
+[![Build Status](https://travis-ci.org/UCLA-CS130/Caviar-Hacker-Grid.svg?branch=master)](https://travis-ci.org/UCLA-CS130/Caviar-Hacker-Grid)
+
 # Caviar-Hacker-Grid
 CS130 Winter 2017 Webserver Project
 
 Commands are to be run from the base Caviar-Hacker-Grid directory.
+# Installation
+You must clone the repository recursively.
+``` 
+git clone --recursive https://github.com/UCLA-CS130/Caviar-Hacker-Grid.git 
+```
```

Comment:
I thought this might be an issue people run into more often than others, but since it's for the instructors I guess it's not needed in this case.